src/internal/observable/bindCallback.ts

```diff
@@ -110,15 +110,20 @@ export function bindCallback(callbackFunc: Function, scheduler?: SchedulerLike):
  *
  * ## Examples
  *
- * ### Convert jQuery's getJSON to an Observable API
+ * ### Convert navigator.geolocation.getCurrentPosition to an Observable API
  * ```ts
  * import { bindCallback } from 'rxjs';
- * import * as jQuery from 'jquery';
  *
- * // Suppose we have jQuery.getJSON('/my/url', callback)
- * const getJSONAsObservable = bindCallback(jQuery.getJSON);
- * const result = getJSONAsObservable('/my/url');
- * result.subscribe(x => console.log(x), e => console.error(e));
+ * // Assuming navigator.geolocation is available on the browser.
+ * const getCurrentPosition = bindCallback(navigator.geolocation.getCurrentPosition);
```

Comment:
This doesn't really work as an example, as `getCurrentPosition` takes the position callback as the first parameter and `bindCallback` wants the caller to:

> Give it a function f of type f(x, callback) and it will return a function g that when called as g(x) will output an Observable.

Also, `getCurrentPosition` also accepts an error callback.

Reply:
Thanks for your feedback... I see your point.

I guess my propossed example works just because _navigator.geolocation.getCurrentPosition_ luckyly works with no parameters. am I right?

Also _navigator.geolocation.getCurrentPosition_ might be continuosly emitting values... so certanly won't match bindCallback purpouse...

So, sorry but i can't think of any other "native" callback which could replace the jquery example :confused:

Reply:
I can't think of one either. Outside of the DOM there is Node's [`fs.exists`](https://nodejs.org/api/fs.html#fs_fs_exists_path_callback), but it is deprecated (and rightly so). I'm happy for this to stay as a jQuery example.

Even if it 'works', there are other issues with `getCurrentPosition`. One of those is that any example would have to make some sense with the TypeScript typing of the bound function and `getCurrentPosition` doesn't make sense, as the callback is not the last parameter.